opm/autodiff/BlackoilModelEbos.hpp

```diff
@@ -1016,12 +1014,15 @@ namespace Opm {
                 const unsigned cellIdx = elemCtx.globalSpaceIndex(/*spaceIdx=*/0, /*timeIdx=*/0);
                 const auto& intQuants = elemCtx.intensiveQuantities(/*spaceIdx=*/0, /*timeIdx=*/0);
                 const auto& fs = intQuants.fluidState();
+                const double pv =
+                    ebosSimulator_.model().dofTotalVolume(cellIdx)
+                    * ebosSimulator_.problem().porosity(cellIdx);
```

Comment:
Also: did we have a clear answer to the question of including variable pore volume (rock compressibility)? I assume including it here would make flow_ebos behave different from flow_legacy?